base.rb

```diff
@@ -72,8 +76,37 @@ class Base < Sinatra::Base
 
   protected
 
+  def monitoring
+    yaml = etcd.get('/monitoring/config').value
+    config = YAML.load(yaml)
+    @monitoring = Tendrl::MonitoringApi.new(config)
+  rescue Etcd::KeyNotFound
+    logger.debug 'Monitoring API not enabled.'
```

Comment:
info seems appropriate here. Administrator would need to know that the API hasn't detected the monitoring API.